opencog/guile/SchemePrimitive.h

```diff
@@ -66,8 +68,39 @@ namespace std {
   /// Alias template index_sequence_for
   template<typename... _Types>
     using index_sequence_for = make_index_sequence<sizeof...(_Types)>;
-}
 
+#elif defined(_LIBCPP_VERSION)
+	// Copied/pasted  from libc++ v1 utility
+	template<class _Tp, _Tp... _Ip>
+	struct _LIBCPP_TEMPLATE_VIS integer_sequence
+	{
```

Comment:
as far as I can tell, `integer_sequence` is not used anywhere at all. Do you really need this?  What happens if it is removed?

Reply:
never mind. I just can't read the code.